packages/app/src/overview/components/overview-screens/pure-overview/overview-content.component.tsx

```diff
@@ -84,8 +84,8 @@ const OverviewContent: FunctionComponent<OverviewContentProps> = ({
       onBackupRestore={onBackupRestore}
     />
     <FileManagerInfo
-      usedSpace={memorySpace.full - memorySpace.free}
-      maxSpace={memorySpace.full}
+      usedSpace={memorySpace.total - memorySpace.reservedSpace}
```

Comment:
Shouldn't it be `memorySpace.total - memorySpace.reservedSpace - memorySpace.usedUserSpace`?

Reply:
We don't use this component anywhere, but I changed it as you suggest, so there will be no confusion. [2f838ad](https://github.com/mudita/mudita-center/pull/915/commits/2f838ad5888146b962ea0c53a4419615d2c45ec8)